src/MoBi.Core/Serialization/Converter/v12/Converter11To12.cs

```diff
@@ -28,10 +30,34 @@ public Converter11To12(Converter110To120 coreConverter)
 
          element.DescendantsAndSelfNamed("MoBiSimulation").Each(simulationNode =>
          {
+            var simulationChanges = simulationNode.DescendantsAndSelfNamed("BuildConfiguration").Any(hasSimulationChanges);
+            if (simulationChanges)
+               simulationNode.AddAttribute("hasUntraceableChanges", true.ToString());
```

Comment:
true to string? lol. what about "true"?

Reply:
yeah... I don't have a better idea but this feels kind of wrong (The property not the boolean)